### PR TITLE
feat: select number of player

### PR DESCRIPTION
**Pull request by thoa01**

### CODE REVIEW

**Review by DanHuy77 — APPROVED** [review]

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js [hunk 1]
```diff
@@ -29,6 +29,8 @@ const JoinRoom = () => {
   }, [location.state?.data]);
   const id_room = location.state?.id_room;
   const currentName = location.state?.name;
+  // let number = location.state?.number;
+// console.log('nu,mber'+number);
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js]: Remove unused code.

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js [hunk 2]
```diff
@@ -83,6 +85,11 @@ const JoinRoom = () => {
       handleKick(currentName);
     }
   };
+  const handleSelectChange = (event) => {
+    let value = event.target.value;
+    const response = axios.post(ip + `user/play/${value}/${id_room}`);
+
+  };
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js]: Add space before and after a function to improve readability.
Remove excessive empty line inside the function.

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js [hunk 3]
```diff
@@ -104,7 +111,8 @@ const JoinRoom = () => {
             <h4 className="jr-left-player">PLAYERS</h4>
             <div className="jr-user">
               <span className="jr-choice-number">
-                <select name="" id="">
+                {role == 1? (
+                  <select onChange={handleSelectChange} name="" id="">
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js:118]: Select tag and option tags inside should have different indents.
Also fix the player number of the option of value 20.

File: the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js [hunk 4]
```diff
@@ -120,6 +128,14 @@ const JoinRoom = () => {
                   <option value="30">30 PLAYERS</option>
                   <option value="50">50 PLAYERS</option>
                 </select>
+                )
+                :(
+                  // <select disabled={false} name="" id="">
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js]: Remove select tag

File: the-telephone-app-fe/src/components/StartGame/StartGame.js [hunk 5]
```diff
@@ -29,10 +29,12 @@ const StartGame = () => {
   };
   const { id_room } = useParams();
   var client = null;
+
   const onConnected = (id_room, data, role) => {
     client.subscribe("/topic/" + id_room, function (response) {
       data = JSON.parse(response.body);
-      navigate("/lobby", { state: { data, id_room, role, name } });
+
+      navigate("/lobby", { state: { data, id_room, role, name} });
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/StartGame/StartGame.js]: Add space after 'name' for this and the below navigates of this file.

File: the-telephone-app-fe/src/components/StartGame/StartGame.js [hunk 6]
```diff
@@ -42,6 +44,12 @@ const StartGame = () => {
           turn = 1;
           navigate("/lobby", { state: { data, id_room, role, name } });
         }
+
+        if (startGame === "MAX") {
+          console.log(data);
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/StartGame/StartGame.js]: Remove console.log when push code to remote.

File: the-telephone-app-fe/src/components/StartGame/StartGame.js [hunk 7]
```diff
@@ -85,9 +91,23 @@ const StartGame = () => {
   const handleJoinClick = async () => {
     const response = await axios.post(ip + `user/join/${id_room}/${name}`);
     const users = response.data;
-    var Sock = new SockJS(ip + "gameplay");
-    client = over(Sock);
-    client.connect({}, () => onConnected(id_room, users, 0), onError);
+    console.log(users);
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/StartGame/StartGame.js]: Remove this

File: the-telephone-app-fe/src/components/StartGame/StartGame.js [hunk 8]
```diff
@@ -85,9 +91,23 @@ const StartGame = () => {
...
+    if (Array.isArray(users)) {
+      var Sock = new SockJS(ip + "gameplay");
+      client = over(Sock);
+      client.connect({}, () => onConnected(id_room, users, 0), onError);
+    }else{
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/StartGame/StartGame.js]: Add spaces

**Review by ces-manhnguyen — APPROVED** [review]

**Pull request merged**